test/integration/scan_test.go

```diff
@@ -22,7 +22,6 @@ import (
 	realtime "github.com/checkmarx/ast-cli/internal/commands/scarealtime"
 	"github.com/checkmarx/ast-cli/internal/commands/util"
 	"github.com/checkmarx/ast-cli/internal/commands/util/printer"
-	applicationErrors "github.com/checkmarx/ast-cli/internal/errors"
```

Comment:
you removed this import, this why the linter have issues

Reply:
Fixed